source/AI.cpp

```diff
@@ -2595,6 +2600,22 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
+// Rove; i. e. wander aimlessly around the system
+void AI::Rove(Ship &ship, Command &command) 
+{
+	// If this is a carried ship (like a fighter), or if the ship is disabled or destroyed, don't rove.
+	if(ship.CanBeCarried() || ship.IsDisabled() || ship.IsDestroyed())
+		return;
+
+	// Create a random point and move to it.
+	int areaSize = 10000;
+	Point destination(Random::Real()*areaSize-areaSize/2, Random::Real()*areaSize-areaSize/2);
```

Comment:
This makes a square. You should be making a circle.
```suggestion
	Point destination = Angle::Random().Unit() * (areaSize * sqrt(Random::Real()));
```